barretenberg/cpp/src/barretenberg/honk/flavor/ecc_vm.hpp

```diff
@@ -65,15 +66,15 @@ template <typename CycleGroup_T, typename Curve_T, typename PCS_T> class ECCVMBa
                                  sumcheck::ECCVMLookupRelation<FF>>;
 
     using LookupRelation = sumcheck::ECCVMLookupRelation<FF>;
-    static constexpr size_t MAX_RELATION_LENGTH = get_max_relation_length<Relations>();
+    static constexpr size_t MAX_PARTIAL_RELATION_LENGTH = get_max_partial_relation_length<Relations>();
 
-    // MAX_RANDOM_RELATION_LENGTH = algebraic degree of sumcheck relation *after* multiplying by the `pow_zeta` random
-    // polynomial e.g. For \sum(x) [A(x) * B(x) + C(x)] * PowZeta(X), relation length = 2 and random relation length = 3
-    static constexpr size_t MAX_RANDOM_RELATION_LENGTH = MAX_RELATION_LENGTH + 1;
+    // BATCHED_RELATION_PARTIAL_LENGTH = algebraic degree of sumcheck relation *after* multiplying by the `pow_zeta`
```

Comment:
To avoid duplicating the same comments in every flavor file, I'll write descriptions of "total" versus "partial" in the comments that define the functions. That should be good enough, right? I think anybody who's curious would look there.

Reply:
yes, that makes sense, also did not realise the nuance until now, self note to take ordering into consideration in a name